css/properties/align-content.json

```diff
@@ -494,6 +495,7 @@
           "__compat": {
             "description": "Supported in Grid Layout",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/align-content",
+            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#align-content-property",
```

Comment:
@sideshowbarker this is actually incorrect, since this is the "Supported in Grid Layout" entry. This might be one of pretty few such cases however, I'm not sure the scripts need to take it into account. Never touching `underscored_stuff` entries would be the easiest way to avoid it.

Reply:
> Never touching `underscored_stuff` entries would be the easiest way to avoid it.

Yeah — previously I did actually have the scraper ignoring all `underscored_stuff` entries, but I stopped having it ignore them because I became aware of some cases where that causes some things to be skipped that shouldn’t have been. That is, I think I ran into cases of actual real BCD feature names that contain underscores.